### ISSUE TITLE
Adjust Stats For Equal Values

### ISSUE THREAD

**Issue by codingwithmanny** [codingwithmanny/devdaorare#29]
**Context:**
If an attribute occurrence is equal to another, the same weighted value should be given to it.

**Example:**
```
Windows Vista = 53
Ubuntu = 53

# Result
Windows Vista = 1
Ubuntu = 1

# Versus Before
Windows Vista = 2
Ubuntu = 1
```

**Comment by jsgm** [comment]
Here is the list of the traits that aren't being weighted equal:

**clothing** ❌
Platinum Trenchcoat (570) -> Black Hoodie  (570) 

**industry** ❌
Farming (533) -> Environmental  (533) 

**language** ❌
Kotlin (295) -> Julia  (295) 
PHP (339) -> Whitespace  (339) 
Python (319) -> JavaScript  (319) 
COBOL (336) -> TypeScript  (336) 
JavaScript (319) -> Python  (319) 

**location** ❌
Kisumu (442) -> Omaha  (442) 
Shenzhen (463) -> Pyongyang  (463) 

**mind** ✅

**os**
Slackware Linux (587) -> Linux Mint  (587) 
Slackware Linux (587) -> Ubuntu  (587) 
Linux Mint (587) -> Ubuntu  (587) 

**textEditor ✅
vibe ✅**

**Comment by jsgm** [comment]
Solved in PR [#31](https://github.com/codingwithmanny/devdaorare/pull/31)
